pkg/constants/constants.go

```diff
@@ -10,3 +10,9 @@ const (
 	LvlDebug              // 4 - Debug-level logs, tricky logic areas
 	LvlTrace              // 5 - Trace-level logs, detailed troubleshooting context
 )
+
+// Service initialization constants
```

Comment:
This will be expanded soon with all service initialization constants, to make sure we don't have magic numbers/strings